### DISCUSSION TITLE
Milestone: 1000 commits!

### DISCUSSION THREAD

**Discussion by kostmo** [swarm-game/swarm#1335] [General]
:partying_face: 

![image](https://github.com/swarm-game/swarm/assets/261693/61ee6413-56f7-4c57-9f46-5f79e26c29b9)

**Comment by byorgey** [comment]
Oh, I hadn't noticed that!  Huzzah!  Much more impressive now that we've switched to doing a single squash merge for each PR.  The first several hundred commits are probably just me making lots of tiny changes...

**Comment by byorgey** [comment]
Just for fun, I went back to look at the very first commit to this repo, which was amusing :smile: 

https://github.com/swarm-game/swarm/commit/479b10e98685cdc4c7c7f1935b679b6b83819dcf
